src/Components/List/List.jsx

```diff
@@ -131,7 +202,13 @@ export default function List({ items, token }) {
                     checked={checked}
                     disabled={checked}
                   />
-                  <label htmlFor={item.name}>{item.name}</label>
+                  <label
+                    htmlFor={item.name}
+                    className={colorCode(item)[0]}
```

Comment:
I like how you tackled the class names and aria-labels for the colour coding by returning arrays with the desired values. However, the labels would be read out by a screen reader as is, but the label's `aria-label` would be read instead of the actual label. We were thinking adding another element (e.g. a badge) with the days next to the label could be a good idea so the significance of the colours are more obvious, and then the aria-label with the days could be tacked onto that.

Reply:
We fixed this by added badges next to the list item names as suggested 😄  The only snag is that the item name is mentioned twice, but better twice than not at all lol. 

![Screen Shot 2020-11-13 at 12 04 34 AM](https://user-images.githubusercontent.com/61370161/99034940-0792d800-2544-11eb-9712-dda9cb0e93c9.png)

Reply:
This looks so good!! After reading EJ's comments in our PR re: aria-labels, I agree 100%.